design/use_cases/indoor_navigation_use_case.md

```diff
@@ -0,0 +1,14 @@
+# Indoor Navigation
+As a Robot user I want my robot to autonomously navigate to a given location on a given map so that it can help me at that location
+
+## More details
+- Why is this needed?
+   - This is needed for indoor robot navigation in most (all?) cases
+   - Example: a courier robot in a logistics warehouse
+
+- What is the expected user interaction?
+   - The user should be able to specify a map to use and a location on that map for the robot to move to. 
+
+- Are there any non-functional requirements? (build system, tools, performance, etc)
```

Comment:
I would suggest that we have a specific requirement that the system is capable of this 2d mode on a low-end processor -- something like the current turtlebot.

Reply:
Good point. Do you mean the Turtlebot 3? I think we need to be specific about a minimum CPU level, I'll follow up on that.

Reply:
I'm not sure what the exact "starting processor" is, but yes, they are using some fairly low-end processors in the latest turtlebots, and still manage to run the existing navigation system.

Reply:
The map-representation(s) will be one of the driving wheel of the design and impose a de facto computation power requirements.